### PR TITLE
Tidy up logging to be more consistent

### PR DESCRIPTION
**Pull request by smozely**

### CODE REVIEW

File: packages/Manager/src/main/kotlin/nz/govt/eop/geo/GeoJsonQueryManifest.kt [hunk 1]
```diff
@@ -15,12 +13,12 @@ const val MANIFEST_CACHE_KEY = "QUERY_MANIFEST"
 @Component
 class GeoJsonQueryManifest(@Autowired val queries: GeoJsonQueries) {
 
-  @Cacheable(cacheNames = arrayOf(MANIFEST_CACHE_KEY))
+  @Cacheable(cacheNames = [MANIFEST_CACHE_KEY])
```
Comment (smozely) [inline packages/Manager/src/main/kotlin/nz/govt/eop/geo/GeoJsonQueryManifest.kt:16]: Just intellij reccomendation

File: packages/Manager/src/main/kotlin/nz/govt/eop/tasks/TaskHelpers.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+) {
+  logger.debug { startTaskMessage(taskName) }
+  val needsRefresh = checkFunc()
+  if (needsRefresh) {
+    logger.info { startTaskRefreshMessage(taskName) }
+    refreshFunc()
+    logger.info { endTaskRefreshMessage(taskName) }
+  }
+  logger.debug { endTaskMessage(taskName) }
+}
```
Comment (smozely) [inline packages/Manager/src/main/kotlin/nz/govt/eop/tasks/TaskHelpers.kt:27]: Extracting this out might have been a little over the top. But made it easy to be consistent with the logging messages and levels

File: packages/Manager/src/main/kotlin/nz/govt/eop/tasks/MinimumFlowLimitBoundariesGenerator.kt [hunk 3]
```diff
@@ -24,12 +24,7 @@ class MinimumFlowLimitBoundariesGenerator(@Autowired val context: DSLContext) {
   @SchedulerLock(name = "checkMinimumFlowLimitBoundaries")
   @Transactional
   fun checkMinimumFlowLimitBoundaries() {
-    logger.debug { "Start task MinimumFlowLimitBoundariesGenerator" }
-    val needsRefresh = doesDataNeedRefresh()
-    if (needsRefresh) {
-      refresh()
-    }
-    logger.debug { "End task MinimumFlowLimitBoundariesGenerator" }
+    processDataRefresh(logger, "checkMinimumFlowLimitBoundaries", ::doesDataNeedRefresh, ::refresh)
```
Comment (smozely) [inline packages/Manager/src/main/kotlin/nz/govt/eop/tasks/MinimumFlowLimitBoundariesGenerator.kt:27]: Its a kotlin thing that we can't share a constant for these names "checkMinimumFlowLimitBoundaries"

File: packages/Manager/src/main/kotlin/nz/govt/eop/tasks/GeoJsonManifestUpdater.kt [hunk 4]
```diff
@@ -13,9 +14,10 @@ class GeoJsonManifestUpdater(val context: DSLContext, val manifest: GeoJsonQuery
   private val logger = KotlinLogging.logger {}
 
   @Scheduled(fixedDelay = 1, timeUnit = TimeUnit.MINUTES)
+  @SchedulerLock(name = "geoJsonManifestUpdater")
```
Comment (vimto) [inline packages/Manager/src/main/kotlin/nz/govt/eop/tasks/GeoJsonManifestUpdater.kt:17]: @smozely does this prevent more than one running concurrently?

**Review by vimto — APPROVED** [review]

File: packages/Manager/src/main/kotlin/nz/govt/eop/tasks/GeoJsonManifestUpdater.kt (hunk 4 above)
Comment (smozely) [reply packages/Manager/src/main/kotlin/nz/govt/eop/tasks/GeoJsonManifestUpdater.kt:17]: @vimto yeah, it ends up using the shedlock table in Postgres to lock against. And has some magic to make sure things don't get locked forever.

**Pull request merged**